packages/component-library/stories/PopoverStories.elm

```diff
@@ -68,16 +68,16 @@ main =
             div [ style "margin-top" "1.5rem" ]
                 [ Popover.view
                     (Popover.default
-                        |> Popover.withHeading "Default"
+                        |> Popover.withHeading "Arrow start"
```

Comment:
Minor changes made to the elm stories, as to exactly match the react stories

Reply:
Nice.